documentation/Installing-Clear-Containers-on-ClearLinux.md

```diff
@@ -0,0 +1,49 @@
+# Install the Clear Containers runtime on Clear Linux
+
+## Introduction
+
+Clear Containers 2.1 is available in Clear Linux since version **13310**.
+Please ensure that your Clear Linux Host is on this version or newer by running the following commands.
+
+```
+$ sudo swupd check-update
+swupd-client software update checker 3.8.0
+   Copyright (C) 2012-2016 Intel Corporation
+
+Attempting to download version string to memory
+Current OS version: 13050
+There is a new OS version available: 13440
+
+```
+If your version is **13310** or newer, skip updating your host.
```

Comment:
That was my concern exactly. At the risk of sounding like Dupont et Dupond, I'd say even more - "it is not necessary to update just for CC."